examples/Example projects/LedStripDriver_OpenSmart/LedStripDriver_OpenSmart.ino

```diff
@@ -0,0 +1,165 @@
+ #define DEBUG_PRINT(x)     Serial.print (x)
+ #define DEBUG_PRINTLN(x)   Serial.println (x)
+
+
+//configuration LED driver
+#define NUM_LEDS  1   //number of leds (chips) in series
+#define LED_PIN_C D3  //clock pin
+#define LED_PIN_D D4  //data pin
+boolean ledOnOff = false;
+float ledHue = 0;
+float ledSaturation = 0;
+float ledBrightness = 0;
+
+//define relais
+#define RELAIS_PIN D2 //activate relais
+
+
+ChainableLED leds(LED_PIN_C, LED_PIN_D, NUM_LEDS);
+
+//Global variables used for the blink without delay example
+unsigned long previousMillis = 0;
+const unsigned long interval = 1000; //Interval in milliseconds
+
+IPAddress ip;
+
+//Arduino functions
+void setup() {
+  //Enable serial port
+  Serial.begin(115200);
+  DEBUG_PRINTLN("Connect to wifi");
+
+  //Connect to network
+  WiFi.begin("your ssid", "your password");
+  while (WiFi.status() != WL_CONNECTED) { 
+    delay(200); 
+    DEBUG_PRINT(".");
+  }
+  //Print IP address
+  ip = WiFi.localIP();
+  DEBUG_PRINT("IP address: ");
+  DEBUG_PRINTLN(ip.toString());
+    
+  //Start Homey library
+  Homey.begin("Arduino LedStrip");
+  Homey.setClass("light");
+  Homey.addCapability("onoff", setLightOnoff);                    //boolean 
+  Homey.addCapability("dim", setLightDim);                        //number 0.00 - 1.00
+  Homey.addCapability("light_hue", setLightHeu);                  //number 0.00 - 1.00
+  Homey.addCapability("light_saturation", setLightSaturation);    //number 0.00 - 1.00
+
+  //set Led status back to Homey
+  Homey.setCapabilityValue("onoff", ledOnOff); //this will not work, I know now why
+
+  //initialise LED driver library
+  leds.init();
+
+  //relais define
+  pinMode(RELAIS_PIN, OUTPUT);
+  
+  DEBUG_PRINTLN("Setup completed");
+}
+
+void loop() {
+  //Handle incoming connections
+  Homey.loop();
+
+  //This is the 'blink without delay' code
+  unsigned long currentMillis = millis();
+  if (currentMillis - previousMillis > interval) {
+    previousMillis = currentMillis;
+    //(This code will be executed every <interval> milliseconds.)
+
+  }
+}
+
+
+
+void SendColorToLeds(){
+  DEBUG_PRINT("SendColorToLeds: ");
+
+  float tmpHue = mapfloat(ledHue, 0.00, 1.00, 1.00, 0.00);  //changing blue and green
+  //saturation determind then maximum value of brightness
+  float tmpBrightMax = mapfloat(ledSaturation, 0.00, 1.00, 1.00, 0.50);
+  float tmpBrightness = mapfloat(ledBrightness, 0.00, 1.00, 0.00, tmpBrightMax);
+  
+  if(ledOnOff){
+    for (byte i=0; i<NUM_LEDS; i++){
+      leds.setColorHSB(i, tmpHue, ledSaturation, tmpBrightness); //led, hue, saturation, brightness);
+      digitalWrite(RELAIS_PIN,LOW);
+      DEBUG_PRINT(" setOn ");
+    }
+  }else{
+    for (byte i=0; i<NUM_LEDS; i++){
+      leds.setColorHSB(i, tmpHue, ledSaturation, 0.00); //led, hue, saturation, brightness);
+      digitalWrite(RELAIS_PIN,HIGH);
+      DEBUG_PRINT(" setOff ");
+    }
+  }
+
+  DEBUG_PRINT("ledOnOff["); 
```

Comment:
Please don't use the debug functions in a sketch. Try to keep everything "vanilla Arduino" so Serial.print(); style...